source/kraken/apply_disruption.cpp

```diff
@@ -237,14 +237,16 @@ struct add_impacts_visitor : public apply_impacts_visitor {
             }
 
             auto nb_rt_vj = mvj->get_rt_vj().size();
-            std::string new_vj_uri = mvj->uri + ":modified:" + std::to_string(nb_rt_vj) + ":" + impact->disruption->uri;
+            std::string new_vj_name =
```

Comment:
I don't see any reason to change the name

Reply:
After discussion: added a commit to let it blank by default as it is displayed to traveler.
(my plan was to keep the default name like before)